/**
 * 断言模块
 */
const assert = require('assert');

/**
 * assert.doesNotThrow(block[,error][,message]);
 * 断言block函数不会抛出错误
 * 暂时没看明白 留坑
 * */

// TODO assert.doesNotThrow(block[,error][,message])